src/model/guild/partial_guild.rs

```diff
@@ -470,7 +470,8 @@ impl PartialGuild {
     ///
     /// impl EventHandler for Handler {
     ///     fn on_message(&self, _: Context, msg: Message) {
-    ///         if let Some(role) = msg.guild_id().unwrap().get().unwrap().role_by_name("role_name") {
+    ///         if let Some(role) =
+    ///            msg.guild_id().unwrap().get().unwrap().role_by_name("role_name") {
```

Comment:
Just checking, does rustfmt want this to be indented 3 spaces?

Reply:
Every indentation - I tried - that features one space from the comment-start is accepted.